docs/tutorials/time-series.md

```diff
@@ -0,0 +1,766 @@
+# Time Series Analysis with Feldera
+
+This guide covers concepts required to effectively process **time series**
+data with Feldera.  A time series is a sequence of events, such as IoT sensor
+readings or financial transactions, where each event is associated with one or
+more timestamps.  Time series data is inherently dynamic, changing continuously
+as new events arrive in real-time.  Feldera is designed to
+efficiently compute over changing data, making it well-suited for time series
+analytics.
+
+:::tip
+
+Most examples in this guide are available as a pre-packaged demo "Time Series Analysis with Feldera"
+that ships with the
+[Feldera Docker container](https://docs.feldera.com/docker) and the
+[online sandbox](https://try.feldera.com).
+
+:::
+
+**Time series data is not special**.  The most important thing to know about
+time series processing in Feldera is that for most intents and purposes time
+series are handled in the same way as any other input data, in particular:
+
+  * Time series inputs are modeled as SQL tables with [input connectors](/connectors)
+    attached to them.
+
+  * Time series tables support inserts, deletes, and upserts.
+
+  * **All** SQL operators apply to time series and non-time-series data with
+    identical semantics, i.e., given the same inputs, Feldera produces the exact
+    same outputs.
+
+**Feldera optimizes time series processing**.  While Feldera offers the same
+expressive power for time series and non-time-series data, it can evaluate some
+classes of time series queries faster and/or using less memory:
```

Comment:
Could be more precise and stronger. Less memory could also be forever growing memory. I think you want to convey that with some simple annotations and depending on the queries, you can often compute with finite memory.